phoenix-pherf/src/it/java/org/apache/phoenix/pherf/PherfMainIT.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ *   or more contributor license agreements.  See the NOTICE file
+ *   distributed with this work for additional information
+ *   regarding copyright ownership.  The ASF licenses this file
+ *   to you under the Apache License, Version 2.0 (the
+ *   "License"); you may not use this file except in compliance
+ *   with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.apache.phoenix.pherf;
+
+import org.junit.Test;
+
+import static org.junit.Assert.fail;
+
+public class PherfMainIT extends ResultBaseTestIT {
+
+    @Test
+    public void testPherfMain() throws Exception{
+        String[] args = {"-q", "--scenarioFile", ".*prod_test_unsalted_scenario.*" };
+        try {
+            Pherf.main(args);
+        } catch (Exception e) {
+            fail("Test encountered an exception.");
+            throw e;
```

Comment:
yep, will remove the try. I think junit will automatically fail if an exception is thrown. This is just wrong.